tweet_analysis/main.py

```diff
@@ -0,0 +1,104 @@
+from nltk.corpus import twitter_samples
+from nltk.tag import pos_tag
+from nltk.stem.wordnet import WordNetLemmatizer
+from nltk.corpus import stopwords
+from nltk import FreqDist
+from nltk import classify
+from nltk import NaiveBayesClassifier
+from nltk.tokenize import word_tokenize
+import re
+import string
+import random
+
+
+# Performing normalization with lemmatization and removes noise
+def remove_noise(tweet_tokens, stop_words=()):
```

Comment:
Is this cleanup specific to Tweets or can we use this for analyzing other sources as well?

Reply:
This cleanup is mostly specific to tweets; it filters out: Twitter handles, hyperlinks, punctuations, and stop words. However, the function can be used to derive a general functional working. It is, nevertheless recommended, to devise a `remove_noise` function for each use case.

Reply:
Great. I wonder if we can factor out some common logic from this in the future.